shared/common-adapters/drop-file-box.desktop.js

```diff
@@ -0,0 +1,96 @@
+// @flow
+import * as React from 'react'
+import fs from 'fs'
+import logger from '../logger'
+import Icon from './icon'
+import {Box, Box2} from './box'
+import {globalStyles, globalColors} from '../styles'
```

Comment:
I think the preferred way to import this became `import * as Styles from '../styles'` at some point because we always ended up importing a bunch of stuff.